### ISSUE TITLE
Medibots Don't Work

### ISSUE THREAD

**Issue by VernaSaurus** [unitystation/unitystation#4071]
## Description
Medibots don't do anything yet. Of course, I decided to take it upon myself to MAKE them work, I'm only stating this to make sure nobody else mistakenly tries to do it despite it already being done.

**Comment by Ninjaboi8175** [comment]
Good idea to create an issue and stating you're working on it. Makes it clear that it's a missing feature that's in progress by you. Looking forward to seeing them work!

**Comment by VernaSaurus** [comment]
Well I seem to have made a system that lets medibot stabilize you when you go into crit, but that isn't what I want.

**Comment by NoooneyDude** [comment]
I don't believe this is still considered in progress. I will adjust the title.

**Comment by corp-0** [comment]
closed by #10735
